### PR TITLE
Update URL parameter documentation

### LINKED ISSUE #1029: URL Parameters Documentation
I stumbled across this in the testing documentation. It should probably be moved to a separate document and reviewed for accuracy.

### PR DESCRIPTION
**Pull request by ZachTRice**
## Description

Fixes #1029

Create dedicated page for URL parameters. Removed unused URL & debug parameters. Added missing URL parameters and updated parameters, values, and descriptions to match current spec.

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules (if applicable)

## Additional Tasks / Recommendations
- [x] Make actual values and placeholder name values distinctive (use brackets for placeholders?)
- [x] Nest (or better organize) the layer parameters (hidden, opacity, palettes)
- [x] Add in layer multi-palettes

Maybe I should add a value type field (boolean, number, string, etc)?

@nasa-gibs/worldview

### CODE REVIEW

**Comment by mike-mcgann** [comment]
How about styling to tell the difference between literals and placeholder? Square brackets are usually for optional items. 

| Parameter | Value |
|-|-|
| p | **geographic**, **arctic**, or **antarctic**|
| v | *minX,minY,maxX,maxY*|
| l | *layer_id1,layer_id2,...,layer_idN*|
| | *layer_id*<b>(opacity=</b>*value*)

There is also *min* and *max*:

https://worldview.earthdata.nasa.gov/?l=MODIS_Terra_Aerosol(min=0.15,0.155,max=0.445,0.45,squash)

File: doc/url_parameters.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# URL Parameters
+
+| Parameter | Value | Description |
+| --------- | ----- | ----------- |
+| `p` | `geographic`, `arctic` or `antarctic` | Selected projection. |
+| `v` | `[minX,minY,maxX,maxY]` | Extent of the map viewport in units based on the projection selected (degrees for EPSG:4326, meters for others). |
```
Comment (mike-mcgann) [inline doc/url_parameters.md]: 'EPSG:4326' should probably change to 'geographic'

File: doc/url_parameters.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| `l` | `[layer_id1],[layer_id2],[layer_id3]` | Active layer list where `layer_id` is the identifier of the layer as defined in the configuration file. Any number of baselayers or overlays may be specified (separated by a `,`). |
+| `l=[layer_id](hidden)` | `(hidden)` | If present, toggles the visibility of the layer as identified by the `layer_id` in the configuration file. |
```
Comment (mike-mcgann) [inline doc/url_parameters.md]: how about "if present, this layer is appears in the layer list but is not shown on the map" or similar wording that sounds good.

**Comment by ZachTRice** [comment]
@mike-mcgann I made changes based on your comments. Have another look and let me know if you have any more suggestions. Thanks!

**Pull request merged**